### PR TITLE
Do not approve this one

### PR DESCRIPTION
**Pull request by rohit-agg**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,6 @@ phpunit.xml
 # Composer
 composer.lock
 vendor/
-.DS_Store
```
Comment (achalaggarwal-cpi) [inline .gitignore:6]: adsad

File: src/core/CIU_Exceptions.php [hunk 2]
```diff
@@ -24,7 +24,7 @@
  * @author      ExpressionEngine Dev Team
  * @link        http://codeigniter.com/user_guide/libraries/exceptions.html
  */
-class CIU_Exceptions extends CI_Exceptions
+class CIU_Exceptions extends MY_Exceptions
```
Comment (achalaggarwal-cpi) [inline src/core/CIU_Exceptions.php:27]: asdasd

File: .gitignore (hunk 1 above)
Comment (rohit-agg) [reply .gitignore:6]: kdfjdsfsdf

**Review by achalaggarwal-cpi — DISMISSED** [review]

**Review by achalaggarwal-cpi — CHANGES_REQUESTED** [review]
asdsad

**Pull request closed without merging**